src/setup-wp.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 sleep 30
-wp core install --url=$URL --title=$WP_TITLE --admin_user=$WP_ADM --admin_password=$WP_PW --admin_email=$ADM_EMAIL --path=$WP_PATH --allow-root
+# needed to hardcode title - env vars and spaces don't mix well
+wp core install --url=$URL --title="Video Games Magazine" --admin_user=$WP_ADM --admin_password=$WP_PW --admin_email=$ADM_EMAIL --path=$WP_PATH --allow-root
```

Comment:
Ha-ha-ha! 🤣 

Could you move back title to the `.env` file and properly use variables here?